sr_gui_grasp_controller/src/sr_gui_grasp_controller/grasp_controller.py

```diff
@@ -458,22 +453,25 @@ def __init__(self, context):
         sublayout.addWidget(btn_frame)
         subframe.setLayout(sublayout)
 
-        selector_layout = QtWidgets.QHBoxLayout()
-        selector_frame = QtWidgets.QFrame()
-
-        selector_layout.addWidget(QtWidgets.QLabel("Select Hand"))
+        hand_finder = HandFinder()
+        if hand_finder.hand_e_available():
+            selector_layout = QtWidgets.QHBoxLayout()
+            selector_frame = QtWidgets.QFrame()
 
-        self.hand_combo_box = QtWidgets.QComboBox()
+            selector_layout.addWidget(QtWidgets.QLabel("Select Hand"))
 
-        for hand_serial in self.hand_parameters.mapping.keys():
-            self.hand_combo_box.addItem(hand_serial)
+            self.hand_combo_box = QtWidgets.QComboBox()
 
-        selector_layout.addWidget(self.hand_combo_box)
+            hand_parameters = hand_finder.get_hand_parameters()
+            for hand_serial in self.hand_parameters.mapping.keys():
+                self.hand_combo_box.addItem(hand_serial)
+            # TODO(@anyone): adapt so that hand Hs are included as options in combo box
```

Comment:
The plugin works with Hand H, just the hand chooser only works with Hand E. So for the moment it would only work (and is tested) with a single handH system. Seems ok for now, so I'm happy if you're happy :)